pkg/controllers/managementuserlegacy/systemimage/systemimage.go

```diff
@@ -1,122 +0,0 @@
-package systemimage
-
-import (
-	"encoding/json"
-	"fmt"
-
-	"github.com/pkg/errors"
-	"github.com/rancher/rancher/pkg/catalog/manager"
-	cutils "github.com/rancher/rancher/pkg/catalog/utils"
-	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
-	"github.com/rancher/rancher/pkg/project"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/runtime"
-)
-
-var systemProjectLabels = labels.Set(map[string]string{"authz.management.cattle.io/system-project": "true"})
-
-type Syncer struct {
-	clusterName    string
-	projectLister  v3.ProjectLister
-	projects       v3.ProjectInterface
-	systemServices map[string]SystemService
-}
-
-func (s *Syncer) SyncProject(key string, obj *v3.Project) (runtime.Object, error) {
-	if obj == nil || obj.DeletionTimestamp != nil {
-		return nil, nil
-	}
-	return obj, s.Sync()
-}
```

Comment:
Is it expected to remove this function which seems to be related to v3 project? Or should we remove only the logic related to catalog?

Reply:
yeah, that is right - the whole systemimage is useless.